### ISSUE TITLE
Confirmation of discontinued is not possible using only keyboard

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7297]
The confirmation box does not receive keyboard focus, instead focus is on the background page.

![image](https://github.com/dart-lang/pub-dev/assets/3865266/b832f810-2d29-4fda-a647-62d96b998447)

**Comment by sigurdm** [comment]
The dialog should probably be modal. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog#creating_a_modal_dialog
